src/threads.js

```diff
@@ -1174,6 +1171,32 @@ Process.prototype.errorObsolete = function () {
     throw new Error('a custom block definition is missing');
 };
 
+Process.prototype.errorBubble = function (error, element) {
+    // Return a morph containing an image of the elment causing the error
+    // above the text of error.
+    var errorMorph = new AlignmentMorph('column', 5),
+        errorIsNested = isNil(element.world()),
+        errorPrefix = errorIsNested ? `${localize('Inside a custom block')}:\n`: '',
```

Comment:
hmm... I don't understand those funny characters ``` `${` ``` is that some wild new ES12 syntax that I need to catch up with?

Reply:
Oh! It's what they call a "template string". Look at that!! I've just learned something new 😂

Reply:
I LOVE template strings! In my own use, they've pretty much replaced all `a + ' ' + b` with ``${a} ${b}`` in my own use. It's also nice because you don't need to escape quotes. :) 

(Though, of course, you do need to escape \`, but that's pretty rare!)

Reply:
how could I not know about template strings until today?! Of course they're incredibly useful. Thanks for showing me!

Reply:
I never heard of them before, either, but as soon as I saw that they're delimited with backquotes it was clear what they are!  This is a longstanding Lisp feature, not for strings, but for Lisp expressions generally.  Lisp uses comma, though, instead of dollarsign; I guess the JS people thought people would be more likely to put a comma in by accident.

The question is, can you nest them?  I.e., can you put one inside the ${...}?   In Lisp, if you do that, you can say, e.g., ",,(...)" to evaluate the expression two environments out.

Reply:
you can indeed nest them! https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#nesting_templates

The "tags" are kind of neat too, though I don't personally use that much.